tests/globalhelper/reporthelper.go

```diff
@@ -188,3 +188,14 @@ func removeCharactersFromString(stringToFormat string, charactersToRemove []stri
 func formatTestCaseName(tcName string) string {
 	return removeCharactersFromString(tcName, []string{"-", "_", " "})
 }
+
+func ConvertSpecNameToFileName(specName string) string {
+	formatString := specName
+	for _, symbol := range []string{" ", ", ", "-"} {
+		formatString = strings.ReplaceAll(formatString, symbol, "_")
+	}
+
+	formatString = removeCharactersFromString(formatString, []string{","})
```

Comment:
You can remove it and just `return strings.ToLower(removeCharactersFromString(formatString, []string{","}))`